src/Lean/Elab/Tactic/GuardMsgs.lean

```diff
@@ -0,0 +1,133 @@
+/-
+Copyright (c) 2023 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller
+-/
+import Lean.Server.CodeActions.Attr
+-- import Std.Lean.Position
+
+/-! `#guard_msgs` command for testing commands
+
+This module defines a command to test that another command produces the expected messages.
+See the docstring on the `#guard_msgs` command.
+-/
+
+open Lean Parser.Tactic Elab Command
+
+namespace Std.Tactic.GuardMsgs
```

Comment:
Perhaps we should take this opportunity to reconsider this namespace? It does leak out from time to time, and `#guard_msgs` is not a tactic, nor is its usefulness restricted to tactics.

Reply:
Done.